nada_dsl/compiler_frontend_test.py

```diff
@@ -577,7 +577,6 @@ def test_binary_operator_integer_publicinteger(operator, name, ty):
     right_ast = AST_OPERATIONS[inner["right"]]
     assert isinstance(left_ast, LiteralASTOperation)
     assert left_ast.value == -3
-    assert len(left_ast.literal_name) == 32
```

Comment:
Why are we deleting this assert? is it intentional?

Reply:
Yes, now the length of literals is no longer 32, so we can't test that.